spec/policies/user_policy_spec.rb

```diff
@@ -33,6 +33,9 @@
     it { is_expected.not_to permit_action(:update) }
     it { is_expected.not_to permit_action(:edit) }
     it { is_expected.not_to permit_action(:destroy) }
+    it { expect(subject.send(:admin?)).to be false }
```

Comment:
@LucasAnderson1984 Cool, makes sense.

There is an argument that says this is a protected part of the API and thus should not be tested. Ideally you really only need to test the public parts as those are the parts that shouldn't be changing a lot, where as the private parts of internal and subject to potential change.